bin/setup-nova-brands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+################################################################################
+# Script Name: setup-nova-brands.sh
+# Script Location: /srv/provision/extensions/pmc/bin/setup-nova-brands.sh
+#
+# Description:
+#   This script installs all nova themes, so that you can set up all nova themes under
+#   single site installation. Nova theme will be used as the base theme.
+#   You must also make it executable by running the command:
+#     chmod +x setup-nova-brands.sh
+#
+# Usage:
+#   ./setup-nova-brands.sh
+#
+# Created: 7 Sept 2023
+################################################################################
+
+# Navigate to nova theme directory
+cd /srv/www/nova/public_html/wp-content/themes
+
+# Clone all nova brands
```

Comment:
In my local, it give me github permission issue. Would I be missing something here ?

Reply:
Your SSH key isn't in your host machine's keychain. You need to run the `ssh-add` command noted in our installation instructions: https://github.com/penske-media-corp/pmc-vvv/blob/master/docs/installing.md#install-vagrant--vvv